src/components/Error/style.css

```diff
@@ -4,6 +4,7 @@
   border-radius: 6px;
   text-align: left;
   color: #FFFFFF;
+  margin-top: -1rem;
```

Comment:
Why do we have a negative value here? Can we do it without a negative value?

Reply:
Because the theme.step class is giving this component too much margin-top resulting in the error screen being too far apart from the back button.

Reply:
Ok, do you think it's worth reducing the theme.step margin-top and having the title be responsible for pushing itself lower?

Reply:
Yes, maybe we could solve it that way.

Reply:
That's not a straight forward fix, I think we should handle it as part of the design makeover @danielspagnolo is working on. He told me margin-top for title is likely to change.